### ISSUE TITLE
Configure gutentags#statusline separators of the modules in progress

### ISSUE THREAD

**Issue by kiryph** [ludovicchabant/vim-gutentags#150]
I am using the plugin [`lightline`](https://github.com/itchyny/lightline.vim) to configure my status line and it looks like 
![mystatuline-gutentags](https://user-images.githubusercontent.com/371756/31022409-ff8b45e0-a538-11e7-89b6-11f0e8ff5560.png)

As you can see the used square brackets do not fit to my statusline. However, `gutentags` has hardcoded them in `autoload/gutentags.vim` for the modules in progress:
```
let l:gen_msg .= '['.join(l:modules_in_progress, ',').']'
```
It would be great to optionally remove or replace them by a choice of the user.

My ideas for the text are
- `Running ctags…`
- `Updating tags…`
- `Indexing…`
- `Tagging…`

Alternative (unicode) symbols indicating a temporary active process: ◉, ▶︎, ✎, ♨︎, ♺

**Comment by kiryph** [comment]
BTW: the in-source documentation
```viml
" Statusline Functions {{{

" Prints whether a tag file is being generated right now for the current
" buffer in the status line.
"
" Arguments can be passed:
" - args 1 and 2 are the prefix and suffix, respectively, of whatever output,
"   if any, is going to be produced.
"   (defaults to empty strings)
" - arg 3 is the text to be shown if tags are currently being generated.
"   (defaults to 'TAGS')
```
is inaccurate. Args 2 and 3 are simply ignored!

**Comment by ludovicchabant** [comment]
I like the idea of using Unicode characters, so I did just that in my `.vimrc` :)

Also, these issues should be fixed with 4304b67481a13612afc184f5f87e7ba70abbbc0c.

**Comment by kiryph** [comment]
Thanks for your update of gutentags.

Unicode characters are nice. However, I always struggle a little bit to pick one which is self-explanatory in the statusline. Context can be helpful. Right now I use the following

- arglist `'⚒ ' .argid_str.'/'.argc()` (if `argc() !=0`)
- shiftwidth `'⇆ '.&shiftwidth`
- tabstop `'↹ '.&tabstop` (if `noet`/`noexpandtab`)
- textwidth `'⟷  '.&textwidth` (if `&textwidth != 0`)
- spelllang `'⚐ '.&spelllang` (if `&spell != 0`)
